internal/api/option/auth.go

```diff
@@ -260,17 +259,19 @@ func (a *authInterceptor) WrapStreamingHandler(
 
 func (a *authInterceptor) listServiceAccounts(
 	ctx context.Context,
-	username string,
-	groups []string,
+	c claims,
 ) (map[string]map[types.NamespacedName]struct{}, error) {
 	queries := []libClient.MatchingFields{
 		{
-			kubeclient.ServiceAccountsBySubjectIndexField: username,
+			kubeclient.ServiceAccountsByOIDCEmailIndexField: c.Subject,
```

Comment:
Is this a bug? Should this be c.Email?

Reply:
😓 

Nice catch.

Reply:
Fixed in b5ad67caa1c15c894428e63020fe1bcdee1093b5